doc/install/mne_python.rst

```diff
@@ -187,8 +187,15 @@ Python development are:
   install Anaconda, and can be started from a terminal with the command
   ``spyder`` (or on Windows or macOS, launched from the Anaconda Navigator GUI).
   If you use Spyder, you should *not* install it into the ``mne`` environment;
-  instead, launch Spyder from the ``base`` environment and set it to use the
-  ``mne`` environment automatically, by opening Spyder and navigating to
+  instead, install ``spyder-kernels`` in the ``mne`` environment and launch
```

Comment:
I'd add some explanation for why it is not the best idea to have mne and Spyder in the same env. For example something like "(this can lead to dependency conficts especially when mne was installed with 3D plotting dependencies)".

Reply:
BTW my recent experience is that trying to follow this instruction and set up Spyder and mne in separate envs can be much more frustrating than problems with dependency conflicts; for example - Spyder won't be able to show mne variables if you double click them if you don't have mne installed in Spyder env (this is the same for variables specific to any other library). So if you wan't to use the variable explorer without frequent errors you would have to mirror the many packages between you mne and spyder envs...
Installing the correct version of spyder-kernels is also not trivial - you would have to check the Spyder docs for the correct mapping of Spyder version -> spyder-kernels version or read the error you get in Spyder if you install the wrong spyder-kernels.

Reply:
and one other thought - Spyder devs seem to recommend installing from `conda-forge` and this can also lead to problems if Spyder is installed in the base env. So maybe it would be better to have three envs: base, mne and spyder, but that gets even more cumbersome...

Reply:
Personally, I don't use `conda` and install with `pip`, but yes, the beginner-friendly approach would be to have both `Spyder` and `MNE` in the same env. Maybe we can split the explanation into beginner (+ warning on what might break, e.g. 3D plotting conflict between dep.) and advanced.

So that's why my variable explorer broke non-stop recently!! Ahah, never thought it was a version difference between dependencies in my base environment and the MNE (or other) venv I'm using! I had so many frustrating episodes recently with Spyder, that I am more and more considering changing my IDE.. Most recent one is the static analysis build-in the editor breaking randomly..

Reply:
Haha, yes, my experiences with Spyder are also bitter-sweet, but I still prefer Spyder to other IDEs. And whenever I have problems and report them on github issues I get a lot of help from the devs. :)